<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="title_wg" name="title_widget" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="login_page">
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>651</width>
         <height>471</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>用户登录</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>用户名</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_account">
                <property name="echoMode">
                 <enum>QLineEdit::Normal</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>密码</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_password">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>记住密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="reg_button">
                <property name="text">
                 <string>注册</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="login_button">
                <property name="text">
                 <string>登录</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="reg_page">
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>501</width>
         <height>561</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>430</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>注册完成</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>120</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_username">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>160</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>210</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>250</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>电话</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>280</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>密保</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>320</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>权限</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_password_reg">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>150</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mibao">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>290</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_phonenumber">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>250</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_sex">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>200</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_power">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>320</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>title_wg</class>
   <extends>QWidget</extends>
   <header location="global">title_wg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
